common/scala/src/main/resources/reference.conf

```diff
@@ -27,7 +27,7 @@ whisk.spi {
   EntitlementSpiProvider = org.apache.openwhisk.core.entitlement.LocalEntitlementProvider
   AuthenticationDirectiveProvider = org.apache.openwhisk.core.controller.BasicAuthenticationDirective
   InvokerProvider = org.apache.openwhisk.core.invoker.InvokerReactive
-  InvokerServerProvider = org.apache.openwhisk.core.invoker.DefaultInvokerServer
+  InvokerServerProvider = org.apache.openwhisk.core.invoker.InvokerServer
```

Comment:
This should be reverted.

Reply:
`DefaultInvokerServer` is already reverted